docs/sdks/dart/GETTING_STARTED.md

```diff
@@ -0,0 +1,31 @@
+## Getting Started
+
+[Add Appwrite](https://github.com/appwrite/sdk-for-dart) SDK to your project’s **pubspec.yaml** dependencies.
+
+```yaml
+dependencies:
+  Dart_appwrite: ^0.1.0
+```
+
+Then get the dependencies by running
+```bash
+dart pub get
+```
+
```

Comment:
This all section is already included in the installation section of every SDK, we can skip it